.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ workflows:
           name: 🚀👩‍💻🇨🇦 deploy-dev-ca2
           stage: dev
           place: ca2
-          context: org-global-v2
+          context: static-cdn-deployer-dev-ca2
```

Comment:
```suggestion
          context: static-cdn-deployer-dev-ca
```

Reply:
and while your are at it, change the name of the deploy job to dev-ca as well

Reply:
We'll probably need to change the `place: ca2` to `place: ca` as well for all occurence of `ca2`